app/controllers/user_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UserController < ApplicationController
-  def index; end
+  def index
+    @users = User.first(5)
+  end
 
```

Comment:
Please use the correct association to the User model in the Post model. It should be `belongs_to :user` not users as a post belongs to only one user not many. I had an association error in the console when was trying to create a post object.
Please make corrections same as in the Post model to `Like`, and `Comment`models

Reply:
please I do not understand as my schema table names are users, likes, comments and posts

Reply:
Hi, so please try to create a post in Rails console and the see the error. You will get to know that that you need to reference your model not schema when you will follow the error message!